src/users/context/users.tsx

```diff
@@ -104,14 +104,8 @@ export const UsersProvider: FC<Props> = React.memo(({children}) => {
         throw new Error(inviteResp.data.message)
       }
 
-      const users =
-        userResp.data?.users?.map(u => ({
-          ...u,
-        })) ?? []
-      const invites =
-        inviteResp.data?.invites?.map(i => ({
-          ...i,
```

Comment:
removed the spreading since it was added to set individual loading states on stuff and it's no longer being used